site/docs/axis.md

```diff
@@ -38,64 +38,44 @@ Axis properties can be customized by setting `axis` to an axis property object.
-| tickColor		| String 		| The color of the axis's tick. |
-| tickLabelColor| String 		| The color of the tick label. |
-| tickLabelFont	| String 		| The font of tick label (e.g., `Helvetica Neue`). |
-| tickLabelFontSize	| Number 		| The font size of label, in pixels. <span class="note-line">__Default value:__ 10. </span> |
-| tickPadding   | Number        | The padding, in pixels, between ticks and text labels.|
-| tickSize      | Number        | The size, in pixels, of major, minor and end ticks. <span class="note-line">__Default value:__  derived from [axis config](config.html#axis-config)'s `tickSize` (`6` by default). </span>|
-| tickSizeMajor | Number        | The size, in pixels, of major ticks.|
-| tickSizeMinor | Number        | The size, in pixels, of minor ticks.|
-| tickSizeEnd   | Number        | The size, in pixels, of end ticks.|
-| tickWidth   	| Number        | The width, in pixels, of ticks.|
-| values        | Number[] &#124; [DateTime[]](transform.html#datetime) | Explicitly set the visible axis tick values. |
-
-### Title
-
-| Property      | Type          | Description    |
-| :------------ |:-------------:| :------------- |
-| title         | String        | A title for the axis. <span class="note-line">__Default value:__  derived from the field's name and transformation function applied e.g, "field_name", "SUM(field_name)", "BIN(field_name)", "YEAR(field_name)".</span> |
-| titleColor	| String 		| The color of the title. |
-| titleFont		| String 		| The font of title (e.g., `Helvetica Neue`). |
-| titleFontWeight	| Number 	 	|  The font weight of title (e.g., `bold`). |
-| titleFontSize	| Number 		| The font size of title, in pixels. <span class="note-line">__Default value:__ 10. </span> |
-| titleOffset   | Number        | The offset (in pixels) from the axis at which to place the title.|
-| titleMaxLength  | Integer     | Max length for axis title when the title is automatically generated from the field\'s description. <span class="note-line">__Default value:__  automatically determined based on the cell size (`config.cell.width`, `config.cell.height`) and `characterWidth` property.</span> |
-| characterWidth  | Integer       | Character width for automatically determining the value of `titleMaxLength`. <span class="note-line">__Default value:__  derived from [axis config](config.html#axis-config)'s `characterWidth` (`6` by default). </span> |
+### Axis Properties
+{% assign schema = site.data.vega-lite-schema.definitions %}
+
+#### General
+
+{% include table.html props="domain,orient,offset,position,zindex" source= schema.Axis.properties %}
```

Comment:
Is there any other pattern besides `schema.XXX.properties`?

If not, make your script only take `XXX` (e.g., `Axis` in this case)?